log-engine-tuulbox/build.gradle.kts

```diff
@@ -14,10 +14,13 @@ kotlin {
         publishAllLibraryVariants()
     }
     js().browser()
+
     iosX64()
     iosArm32()
     iosArm64()
     macosX64()
+    macosArm64()
+    iosSimulatorArm64()
```

Comment:
Have you tested that BLE is available on the simulator?

Reply:
As above, not yet.

Reply:
Removed!